src/pages/HomePage.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+function HomePage() {
+    let user = JSON.parse(localStorage.getItem("user"));
+    const alert = useSelector(state => state.alert);
+    return (
+        <div>      
+            {alert.message} : {user.user.email}
+        </div>
+    )
+}
```

Comment:
```suggestion
function HomePage() {
    const { user: { email }} = JSON.parse(localStorage.getItem("user"));
    const { message } = useSelector(state => state.alert);
    return (
        <div>      
            {message} : {email}
        </div>
    )
}
```

I know this will be removed, just a suggestion for the feature